it/v2/endpoints/AmendForeignPropertyPeriodSummaryControllerISpec.scala

```diff
@@ -346,16 +346,50 @@ class AmendForeignPropertyPeriodSummaryControllerISpec extends V2IntegrationBase
       """.stripMargin
   }
 
+
+  private trait NonTysTest extends Test {
+    def taxYear: String                            = "2022-23"
+    def downstreamTaxYear: String                  = "2022-23"
+    def downstreamQueryParams: Map[String, String] = Map("taxYear" -> downstreamTaxYear) ++ commonQueryParams
+
+    override def downstreamUri: String = s"/income-tax/business/property/periodic"
```

Comment:
Can we factor out `"/income-tax/business/property/periodic"` and replace it with `baseUri` and then add `def baseUri: String = s"/income-tax/business/property/periodic"` in the test trait. `line 363` can also then be updated to be `override def downstreamUri: String = baseUri + s"/$downstreamTaxYear"` . I think it makes it easier to understand the difference in the downstreamUri between non-TYS and TYS